### ISSUE TITLE
Fullscreen image / movie do not stand still

### ISSUE THREAD

**Issue by jimi3** [telegramdesktop/tdesktop#10599]
### Steps to reproduce
1. click on picture or movie to go fullscreen

### Expected behaviour
no wobbling, no up and down, just a normal view of a picture

### Actual behaviour
Picture / Movie wobbles goes up/down fast and never stays still

### Configuration
Linux snap install

**Version of Telegram Desktop:**
2.7.1

**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:
snap

**Used theme**:
tested them all, wobbles fast on any theme / dark mode

**Comment by Linfty** [comment]
I have the same problem. It has started since 2.6.2 version. Version 2.6.1. works fine. I think it is depend on desktop environment. I use Xmonad.

**Comment by ilya-fedin** [comment]
Yeah, there are a problem with Xmonad. Looking by their issue tracker even developers don't know how to force it to not to tile tdesktop, so we awaiting when they suggest something
https://github.com/xmonad/xmonad/issues/279

**Comment by burunduk3** [comment]
I workarounded this by adding `title =? "Media viewer" --> doFloat` into manageHook.

**Comment by Linfty** [comment]
> I workarounded this at xmonad's level, by adding `title =? "Media viewer" --> doFloat` into manageHook in the config.

Yes, it works. Thanks a lot

**Comment by jimi3** [comment]
> Can you provide more info about your system?

sorry for late reply, running default Qubes OS 4.0.4 Desktop which comes with xfce and telegram-desktop runs in a fedora 32 VM with fusion repo. 
Clueless how to apply above fix to my System.

**Comment by ilya-fedin** [comment]
> xfce

Can't reproduce with Manjaro Xfce 4.16 and tdesktop 2.7.1 static binary from desktop.telegram.org

**Comment by jimi3** [comment]
defaulted to fedoras 33 dnf package which was 2.6.x had no wobbling, they did update though now to 2.7.1 and the wobbling is back
